ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -2840,6 +2845,11 @@ tests:
   steps:
     cluster_profile: gcp-qe
     env:
+      COMPUTE_CONFIDENTIAL_COMPUTE: AMDEncryptedVirtualizationNestedPaging
```

Comment:
Suggest to shorten the ci job name to `gcp-ipi-secureboot-confidential-computing-mini-perm-f14`, them move it next to `gcp-ipi-confidential-computing-fips-mini-perm-f28-destructive`

Reply:
And remove the following `gcp-ipi-secureboot-mini-perm-f28-destructive` one ?